setup.py

```diff
@@ -36,5 +36,6 @@
       },
       install_requires = ['Twisted[tls]',
                           'service-identity',
-                          'arrow'],
+                          'arrow',
+                          'configparser'],
```

Comment:
configparser is included with Python 3 and so this shouldn't be needed. I haven't tested ceph_exporter with Python 2, does this mean you have? I'm not against supporting Python 2, but this requirement should be made conditional so that it doesn't get pulled in when using Python 3.